tensorflow/core/kernels/dml_broadcast_to_op.cc

```diff
@@ -1,12 +1,9 @@
 /* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
 Portions Copyright (c) Microsoft Corporation.
```

Comment:
Copyright banner shouldn't be modified.

Reply:
I'm super confused, I have no idea why this got modified.